powershell/llcsharp/schema/string.ts

```diff
@@ -398,10 +398,10 @@ ${this.validateEnum(eventListener, property)}
     return `await ${eventListener}.AssertRegEx(${nameof(property.value)},${property},@"${pattern}");`;
   }
   private validateEnum(eventListener: Variable, property: Variable): string {
-    if (!(this.schema instanceof ChoiceSchema)) {
+    if (this.schema.type !== SchemaType.Choice && this.schema.type !== SchemaType.SealedChoice) {
```

Comment:
should just this.schema.type !== SchemaType.SealedChoice

Reply:
Got it. Will fix in next PR